js/src/product-feed/submission-success-guide/enhanced-conversion/footer.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
+import AppButton from '.~/components/app-button';
+import CTA from '.~/components/enhanced-conversion-tracking-settings/cta';
+
+const Footer = ( { handleGuideFinish } ) => {
+	const { createNotice } = useDispatchCoreNotices();
+
+	const handleEnableOrDisableClick = useCallback( () => {
+		createNotice(
+			'info',
+			__( 'Status successfully set', 'google-listings-and-ads' )
+		);
+
+		handleGuideFinish?.();
+	}, [ createNotice, handleGuideFinish ] );
+
+	return (
+		<Fragment>
+			<div className="gla-submission-success-guide__space_holder" />
+
+			<AppButton
+				isSecondary
+				// @todo: review data-action label
```

Comment:
@joemcgill Can you advise here please? Thanks!

Reply:
@joemcgill Can you kindly advise please?

Reply:
It is used to indicate the modal is closed with which action.

https://github.com/woocommerce/google-listings-and-ads/blob/8ba1145230befe2520f64bbb0eaf35cddecf3bd1/js/src/product-feed/submission-success-guide/index.js#L156

While it might seem redundant to record an event saying "the modal is closed by close", it's possible to distinguish how the user closed the modal against the UI, so it's recommended to use "close". The `'view-product-feed'` will no longer exist so it would be appreciated if the above `@fires` doc could be updated accordingly.